src/styles/auto-prefix.js

```diff
@@ -28,7 +28,7 @@ export default {
         userAgent: userAgent,
       });
 
-      return prefixer.prefix;
+      return prefixer;
```

Comment:
True, I believe it would. And it is actually better like that since I forgot to change this other code:
    
    if (userAgent === false) { // Disabled autoprefixer
      return (style) => style;
    } else if (userAgent === 'all' || userAgent === undefined) { // Prefix for all user agent
      return InlineStylePrefixer.prefixAll;
    ...

Now, what about the *set()* method? It will be removed but the components that use this method are not having their styles transformed right now.